applications/nrf5340_audio/src/bluetooth/bt_management/bt_mgmt.h

```diff
@@ -64,6 +66,10 @@ enum bt_mgmt_scan_type {
  *				device name or broadcast name. Can be max
  *				BLE_SEARCH_NAME_MAX_LEN long; everything beyond that value
  *				will be cropped. Can be NULL. Shall be '\0' terminated.
+ * @param[in]	brdcast_id	Broadcast ID to search for, only valid if @p type is
```

Comment:
Nit: A quick glance at the interwebs shows "brdcast" not being associated often as short form for "broadcast". I suggest "bc_id", "bcast_id", or "bcst_id" as alternatives.

Reply:
Not changed, discussed with the rest of the team and we are happy with that variable name